person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_reader :id
+
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1...99_999)
+    @name = name
+    @age = age
+    @permission = parent_permission
+  end
+
+  def can_use_services?
+    return true if @permission and of_age?
+
+    false
+  end
+
+  private
+
+  def of_age?
+    return true if @age >= 18
+
+    false
```

Comment:
- Optional 
Any function in ruby returns the last line automatically. Here you don't need to explicitly mention the return statement.
Instead, you can do it as below.

def of_age?
    @age >= 18
end

Reply:
Oh, awesome!! I didn't think of this. Thanks for the tips.